esf.tf

```diff
@@ -7,7 +7,7 @@
 
 ###### Elastic Serverless Forwarder
 locals {
-  dependencies-bucket-url = "http://esf-dependencies.s3.amazonaws.com"
+  dependencies-bucket-url = "https://esf-dependencies.s3.amazonaws.com"
```

Comment:
Looks good to me. I will wait for CLA signing and go ahead with this.

cc @constanca-m  fyi to check other concerns